cdap-docs/_common/common-build.sh

```diff
@@ -175,7 +181,10 @@ function build_license_pdfs() {
   version
   cd $SCRIPT_PATH
   PROJECT_VERSION_TRIMMED=${PROJECT_VERSION%%-SNAPSHOT*}
-  rm -rf $SCRIPT_PATH/$LICENSES_PDF
+  if [ -d "${SCRIPT_PATH}/${LICENSES_PDF}" ]
+  then
+    rm -rf $SCRIPT_PATH/$LICENSES_PDF
+  fi
```

Comment:
Because the way JJ did it was wrong. This is probably why the script tried to eat Julien's dad drive one day when he invoked it from the wrong location, as it was probably deleting merrily away without testing first...